web/src/lib/components/album-page/albums-list.svelte

```diff
@@ -257,8 +256,11 @@
     await deleteSelectedAlbum();
   };
 
-  const handleEdit = (album: AlbumResponseDto) => {
-    albumToEdit = album;
+  const handleEdit = async (album: AlbumResponseDto) => {
+    await modalManager.show(EditAlbumForm, {
+      album,
+      onEditSuccess: successEditAlbumInfo,
```

Comment:
Just looked at this again and you should get rid of `onEditSuccess`. EditAlbumForm should have an `onClose(album?: ...) => void`; if album is provided it was a success, otherwise it was cancelled. That's how we treat those generally now